doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -658,7 +658,7 @@ As sysbuild deals with configuration of features for some features and propagati
 Incompatibities
 ===============
 
-In the sysbuild release included in the |NCS| 2.7, the following features of the multi-image builds using child an parent images are not supported:
+In the sysbuild release included in the |NCS| 2.7, the following features of the multi-image builds using child and parent images are not supported:
```

Comment:
Hi @FrancescoSer, this seems like a typo. Can we integrate it to your PR: https://github.com/nrfconnect/sdk-nrf/pull/15760?